cmd/collectors/rest/plugins/health/health.go

```diff
@@ -0,0 +1,189 @@
+package health
+
+import (
+	goversion "github.com/hashicorp/go-version"
+	"github.com/netapp/harvest/v2/cmd/collectors"
+	"github.com/netapp/harvest/v2/cmd/poller/plugin"
+	"github.com/netapp/harvest/v2/cmd/tools/rest"
+	"github.com/netapp/harvest/v2/pkg/conf"
+	"github.com/netapp/harvest/v2/pkg/errs"
+	"github.com/netapp/harvest/v2/pkg/matrix"
+	"github.com/tidwall/gjson"
+	"strings"
+	"time"
+)
+
+type AlertSeverity string
+
+const (
+	errr    AlertSeverity = "error"
+	warning AlertSeverity = "warning"
+)
+
+const diskHealthMatrix = "health_disk"
+const severityLabel = "severity"
+
+type Health struct {
+	*plugin.AbstractPlugin
+	client *rest.Client
+	data   map[string]*matrix.Matrix
+}
+
+func New(p *plugin.AbstractPlugin) plugin.Plugin {
+	return &Health{AbstractPlugin: p}
+}
+
+var metrics = []string{
+	"alerts",
+}
+
+func (v *Health) Init() error {
+
+	var err error
+
+	if err = v.InitAbc(); err != nil {
+		return err
+	}
+
+	if err = v.initAllMatrix(); err != nil {
+		return err
+	}
+
+	timeout, _ := time.ParseDuration(rest.DefaultTimeout)
+	if v.client, err = rest.New(conf.ZapiPoller(v.ParentParams), timeout, v.Auth); err != nil {
+		v.Logger.Error().Stack().Err(err).Msg("connecting")
+		return err
+	}
+
+	if err = v.client.Init(5); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (v *Health) initAllMatrix() error {
+	v.data = make(map[string]*matrix.Matrix)
+	if err := v.initMatrix(diskHealthMatrix); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (v *Health) initMatrix(name string) error {
+	v.data = make(map[string]*matrix.Matrix)
+	v.data[name] = matrix.New(v.Parent+name, name, name)
```

Comment:
may be directly use `diskHealthMatrix` here, so `initMatrix` only needed and we don't need `initAllMatrix`.